src/di.php

```diff
@@ -70,10 +73,6 @@
         }
 
         $log->addWriter($writer2);
-    } else {
-        $logFile = $di['config']['path_logs'];
-        $writer = new Box_LogStream($logFile);
-        $log->addWriter($writer);
```

Comment:
Do we always now log to disk, regardless of log to DB? Or has this check been moved elsewhere?

Reply:
I did that while testing, having them both enabled would be redundant. Reverting that.